db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/StateMachines/VMProvision_VM.class/__methods__/update_provision_status.rb

```diff
@@ -19,3 +19,8 @@
 updated_message += "Current Retry Number [#{$evm.root['ae_state_retries']}]" if $evm.root['ae_result'] == 'retry'
 prov.miq_request.user_message = updated_message
 prov.message = status
+
+if $evm.root['ae_result'] == "error"
+  $evm.create_notification(:level => "error", :subject => prov.miq_request, \
+                           :message => "VM Provision Error: #{updated_message}")
```

Comment:
@billfitzgerald0120 @gmcculloug @tinaafitz 
Since this is a Cloud should the "VM Provision Error:" be "Instance Provision Error" since its going to be displayed to the end user?